tests/sles4sap/ipaddr2/cluster_create.pm

```diff
@@ -2,7 +2,57 @@
 # SPDX-License-Identifier: GPL-2.0-or-later
 
 # Summary: Create cluster
-# Maintainer: QE-SAP <qe-sap@suse.de>, Michele Pagot <michele.pagot@suse.com>
+# Maintainer: QE-SAP <qe-sap@suse.de>
+
+=head1 NAME
+
+ipaddr2/cluster_create.pm - Create the Pacemaker cluster for the ipaddr2 test
+
+=head1 DESCRIPTION
+
+This module is responsible for setting up the Pacemaker cluster on the two
+SUT (System Under Test) virtual machines.
+
+Its primary tasks are:
+
+- If cloud-init was disabled (B<IPADDR2_CLOUDINIT> is 0), it installs the nginx web server
+  on both SUT nodes.
+  This step is necessary because without cloud-init, the web server would not
+  have been pre-installed.
+- Initializes and configures the Pacemaker cluster across the two SUT nodes,
+  preparing them for high-availability resource management.
+
+=head1 VARIABLES
```

Comment:
I still think `VARIABLES` as a name for openQA settings could be confusing. openQA itself doesn't help by showing in the results a `Settings` tab, but also by having a `vars.json` in the `Logs & Assets` tab.